prov/rxm/src/rxm_ep.c

```diff
@@ -168,8 +168,22 @@ static int rxm_recv_queue_init(struct rxm_recv_queue *recv_queue, size_t size,
 
 static void rxm_send_queue_close(struct rxm_send_queue *send_queue)
 {
-	if (send_queue->fs)
+	if (send_queue->fs) {
```

Comment:
@a-ilango This is what you asked for in the previous PR related to RxM
This routine cleans queue and releases tx_buf from the tx_pool

Reply:
Thanks. This is good! 👍